### PR TITLE
Upstream a Chrome Flexbox test

### PR DESCRIPTION
**Pull request by cbiesinger**
This contributes our first flexbox test. This commit also adds some of our
testing infrastructure, especially check-layout-th.js, a script that allows
checking expected dimensions of an element using data-expected-width attributes
(and similar).

[Review on Reviewable](https://reviewable.io/reviews/w3c/csswg-test/980)

### CODE REVIEW

File: css-flexbox-1/percentage-heights-000.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    height: 40%;
+}
+.column {
+    flex-flow: column wrap;
+    width: 100px;
+    height: 100px;
+    align-content: flex-start;
+}
+</style>
+<script src="../../resources/testharness.js"></script>
```
Comment (kojiishi) [inline css-flexbox-1/percentage-heights-000.html]: testharness.js should be `"/resources/testharness.js"`.

File: css-flexbox-1/percentage-heights-000.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+<script src="../../resources/testharnessreport.js"></script>
```
Comment (kojiishi) [inline css-flexbox-1/percentage-heights-000.html]: Same, resources should be in `/resources/`.

File: css-flexbox-1/percentage-heights-000.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+<script src="../../resources/check-layout-th.js"></script>
```
Comment (kojiishi) [inline css-flexbox-1/percentage-heights-000.html]: And this one is `"support/check-layout-th.js"`?

**Comment by cbiesinger** [comment]
@kojiishi Thank you, fixed. By the way, how do I run the tests locally in a way that the /resources references work?

**Comment by Ms2ger** [comment]
Fwiw, I've got a symlink into csswg-test from my wpt clone, so I can use `./serve` there.

**Comment by kojiishi** [comment]
I run `python -m SimpleHTTPServer` on the root directory.

**Comment by kojiishi** [comment]
lgtm.

**Pull request merged**